daemon/config/config.go

```diff
@@ -22,6 +22,9 @@ type Config struct {
 	// Network config
 	NetworkConfg network.Config
 
+	// Whether enable cri manager.
+	CriEnabled bool `json:"cri-enabled,omitempty"`
```

Comment:
I am afraid this field addition introduces some compatibility.

I see two similar fields:

```
CriEnabled bool `json:"cri-enabled,omitempty"`
IsLxcfsEnabled bool `json:"enable-lxcfs,omitempty"`
```

They are quite similar and have different. Could we change into `IsCriEnabled bool `json:"enable-cri,omitempty"``

Reply:
Can't agree more :) be in consistent !